opentracing-spring-rabbitmq-it/src/test/java/io/opentracing/contrib/spring/rabbitmq/RabbitMqTracingAutoConfigurationItTest.java

```diff
@@ -49,6 +55,7 @@ public void givenAutoConfiguredRabbitTemplate_sendAndReceive_shouldBeTraced() {
     Message response = rabbitTemplate.sendAndReceive("myExchange", "#", requestMessage, null);
 
     assertConsumerAndProducerSpans(0);
```

Comment:
actually the line above should have been in this commit (didn't notice it when splitting my changes into 2 commits)

Reply:
ah, never mind my last comment - everything is correct here (as previously the TestMessageListener didn't send a reply so that actually count of spans and their tags were correct also before this change)